pkgs/top-level/config.nix

```
@@ -131,10 +143,26 @@ let
     checkMeta = mkOption {
       type = types.bool;
       default = false;
-      description = ''
+      description = lib.mdDoc ''
         Whether to check that the `meta` attribute of derivations are correct during evaluation time.
       '';
     };
+
+    packageOverrides = mkOption {
+      default = const {};
+      type = overrideFnType;
+      description = lib.mdDoc ''
+        **Deprecated.** Use `overlays` instead!
+      '';
+    };
+
+    perlPackageOverrides = mkOption {
+      default = const {};
+      type = overrideFnType;
+      description = lib.mdDoc ''
+        **Deprecated.** Use `overlays` instead!
```

I couldn't find documentation that describes an alternative. I also try to keep a safe distance from perl, but a search for `perlPackageOverrides` should have been sufficient to find such docs. `perl.section.md` does not mention `override` or `overlay` either.

```suggestion
        A function that takes the final set of Perl packages and returns an attribute set of packages to add or change.
```

---

I only added it because it was in the nixpkgs module. The only other reference I could find was in the perl expression itself.